syntax = "proto3";
package plugin_version;





/*
CreatePluginVersion请求
*/
message CreatePluginVersionRequest {
    /*
    插件Id
    */
    string id = 1 ;
    /*
    版本名称
    */
    string name = 2 ;
    /*
    备注
    */
    string memo = 3 ;
}

/*
CreatePluginVersion返回
*/
message CreatePluginVersionResponse {
    /*
    ID
    */
    string id = 1 ;
}

/*
CreatePluginVersionApi返回
*/
message CreatePluginVersionResponseWrapper {
    /*
    返回码
    */
    int32 code = 1 ;
    /*
    返回码解释
    */
    string codeExplain = 2 ;
    /*
    错误详情
    */
    string error = 3 ;
    /*
    返回数据
    */
    plugin_version.CreatePluginVersionResponse data = 4 ;
}

